<?xml version="1.0"?>
<launch>
    <node pkg="mdr_handle_open_action" type="handle_open_action" name="handle_open_server" output="screen">
        <param name="move_arm_server" value="move_arm_server" />
        <param name="move_base_server" value="move_base_server" />
        <param name="gripper_controller_pkg_name" value="mas_hsr_gripper_controller" />
        <param name="pregrasp_config_name" value="neutral" />
        <param name="safe_arm_joint_config" value="folded" />
        <param name="handle_open_dmp" value="$(find mdr_place_action)/config/trajectory_weights/weights_table_place.yaml" />
        <param name="dmp_tau" value="1" />
    </node>
</launch>
